### PR TITLE
[#142] Improve Airflow start up speed

### LINKED ISSUE #142: Improve Airflow start up speed
Airflow web services start takes quite long (about 2-3 mins) this causes issues during deployment since airflow api is not available during post deployment tests. One of the possible starting points for investigation might be the next startup step which takes about 30 second for each worker:
`
[2018-05-29 11:13:00,122] {{models.py:189}} INFO - Filling up the DagBag from /airflow-dags
[2018-05-29 11:13:00,136] {{models.py:189}} INFO - Filling up the DagBag from /airflow-dags
/usr/local/lib/python3.5/dist-packages/flask/exthook.py:71: ExtDeprecationWarning: Importing flask.ext.cache is deprecated, use flask_cache instead.
  .format(x=modname), ExtDeprecationWarning
[2018-05-29 11:13:30 +0000] [51] [INFO] Handling signal: ttin
`

### PR DESCRIPTION
**Pull request by alexander-semenets**
this PR resolves #142 
What's been done:
- decreased airflow worker refresh interval
- updated airflow webserver readiness probe
- removed hardcoded timeout during test stage
- added debug mode for further purposes

### CODE REVIEW

**Comment by alexander-semenets** [comment]
ok to test

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
@akharlamov could you please approve the change

File: deploy/ansible/roles/legion_enclave/templates/airflow-values.yaml.j2 [hunk 1]
```diff
@@ -28,11 +28,14 @@ postgres:
   port: 5432
 
 core:
-  logging_level: info
+  logging_level: debug
```
Comment (kirillmakhonin) [inline deploy/ansible/roles/legion_enclave/templates/airflow-values.yaml.j2]: Maybe we should disable debbuging back?

File: deploy/full-lifecycle.Jenkinsfile [hunk 2]
```diff
@@ -7,7 +7,7 @@ node {
             result = build job: params.BuildLegionJobName, propagate: true, wait: true, parameters: [
                     [$class: 'GitParameterValue', name: 'GitBranch', value: params.GitBranch],
                     booleanParam(name: 'PushDockerImages', value: true),
-                    booleanParam(name: 'EnableDockerCache', value: false),
+                    booleanParam(name: 'EnableDockerCache', value: true),
```
Comment (kirillmakhonin) [inline deploy/full-lifecycle.Jenkinsfile]: You can not use this parameter now, it is obsollete

File: deploy/helms/airflow/values.yaml [hunk 3]
```diff
@@ -25,7 +25,7 @@ postgres:
   port: 5432
 
 core:
-  logging_level: warning
+  logging_level: debug
```
Comment (kirillmakhonin) [inline deploy/helms/airflow/values.yaml]: Here too

File: deploy/ansible/roles/legion_enclave/templates/airflow-values.yaml.j2 (hunk 1 above)
Comment (alexander-semenets) [reply deploy/ansible/roles/legion_enclave/templates/airflow-values.yaml.j2]: don't thinks we really suffer from debug, the output is not so verbose but it helps to investigate some issues on the flight without the need to reconfigure pods and redeploy.

File: deploy/full-lifecycle.Jenkinsfile (hunk 2 above)
Comment (alexander-semenets) [reply deploy/full-lifecycle.Jenkinsfile]: I see the parameter implemented across all our pipelines and jobs, so guess it worth to refactor them in a separate story, just made a proper use of that for the purpose of the build process here

File: deploy/helms/airflow/values.yaml (hunk 3 above)
Comment (alexander-semenets) [reply deploy/helms/airflow/values.yaml]: the same as above

**Pull request merged**